docs/reference/regions.md

```diff
@@ -9,15 +9,18 @@ When you create a cluster in Camunda 8 SaaS, you must specify a region for that
 Currently, we make these regions available for customers on the Trial, Starter, and Enterprise Plans. Enterprise customers can discuss custom regions with their Customer Success Manager.
 
 :::note
-Our Console and Web Modeler components are currently hosted in the EU. [Contact us](/reference/contact.md) if you have additional questions.
-:::
 
-Below, find a list of regions currently supported in Camunda 8 SaaS.
+- Management cluster components (Console and Web Modeler) are currently hosted in GCP (EU). [Contact us](/reference/contact.md) if you have additional questions.
```

Comment:
What do you think about calling out here also the fact that the single-tenant clusters can run on a dedicated infrastructure on GCP or AWS? This way it sets the scene for the below list of supported regions.